### PR TITLE
SUS-4812 | check opt-in status in the internal tracker 🇪🇺

### PR DESCRIPTION
**Pull request by mszabo-wikia**
https://wikia-inc.atlassian.net/browse/SUS-4812

### CODE REVIEW

File: extensions/wikia/Track/templates/track.mustache [hunk 1]
```diff
@@ -1,69 +1,77 @@
...
+					'&session_id=' + window.sessionId + '&pv_unique_id=' + window.pvUID + '&pv_number=' + window.pvNumber +
+					'&pv_number_global=' + window.pvNumberGlobal;
 
-	script.src = trackUrl;
-	document.head.appendChild(script);
-})();
+			if (optIn) {
+				trackUrl += '&u=' + window.wgTrackID;
+			} else {
+				trackUrl += '&u=0';
```
Comment (rogatty) [inline extensions/wikia/Track/templates/track.mustache]: It should be `u=-1`

File: skins/oasis/modules/templates/Oasis_Index.php [hunk 2]
```diff
@@ -65,6 +65,8 @@
 	<div class="background-image-gradient"></div>
 <? endif ?>
 
+<?= $internalTracker ?>
```
Comment (rogatty) [inline skins/oasis/modules/templates/Oasis_Index.php:68]: Shouldn't this be loaded after `<?= $trackingOptIn ?>`? I'm not sure what are the dependencies.

Comment (mszabo-wikia) [reply skins/oasis/modules/templates/Oasis_Index.php:68]: the module proper is loaded in the head, the code below will init it to show the modal, so we can lazy queue the tracker as callback to be fired after the user's consent status has been obtained

File: extensions/wikia/Track/templates/track.mustache (hunk 1 above)
Comment (mszabo-wikia) [reply extensions/wikia/Track/templates/track.mustache]: addressed in 0f77ce8bd3f

**Review by rogatty — APPROVED** [review]

**Pull request merged**